config/database.example.yml

```diff
@@ -9,7 +9,7 @@ default: &default
   encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   username: postgres
-  password: postgres
+  #password: postgres ##Uncomment This
```

Comment:
Is this change required?

Reply:
I got password errors as Appveyor has Postgresql Password 'Password12!' , and as we copy the example database config it uses postgres as the password